.github/workflows/update-gardenlogin.yaml

```diff
@@ -2,63 +2,152 @@ name: gardenlogin-updater
+            --clobber
+
+      # ───── GitHub‑App token (needed for repo‑dispatch) ──────────────────────
       - name: Get token for gardener-github-pkg-mngr app
-        if: github.event.release.prerelease == false
+        if: (github.event_name == 'workflow_dispatch' && inputs.dry_run == false) || (github.event_name == 'release' && github.event.release.prerelease == false)
         id: gardener-github-workflows
-        uses: getsentry/action-github-app-token@d4b5da6c5e37703f8c3b3e43abb5705b46e159cc # v3.0.0
+        uses: actions/create-github-app-token@df432ceedc7162793a195dd1713ff69aefc7379e   # v2.0.6
         with:
-          app_id: ${{ secrets.GARDENER_GITHUB_WORKFLOW_PKG_MNGR_APP_ID }}
-          private_key: ${{ secrets.GARDENER_GITHUB_WORKFLOW_PKG_MNGR_APP_PRIVATE_KEY }}
-      - name: Send update with latest versions to ${{ github.repository_owner }}/homebrew-tap
-        if: github.event.release.prerelease == false
+          app-id:      ${{ secrets.GARDENER_GITHUB_WORKFLOW_PKG_MNGR_APP_ID }}
+          private-key: ${{ secrets.GARDENER_GITHUB_WORKFLOW_PKG_MNGR_APP_PRIVATE_KEY }}
+          owner:       ${{ github.repository_owner }}
+          repositories: |
+            homebrew-tap
+            chocolatey-packages
+
+      # ───── SHA‑256 sums (all in one pass) ───────────────────────────────────
+      - name: Compute checksums
+        if: github.event_name != 'workflow_dispatch' || (inputs.skip_build == false  && inputs.dry_run == false)
+        run: |
+          set -euo pipefail
+          echo "DARWIN_SHA_AMD64=$(sha256sum bin/darwin-amd64/gardenlogin_darwin_amd64      | awk '{print $1}')" >> "$GITHUB_ENV"
+          echo "DARWIN_SHA_ARM64=$(sha256sum bin/darwin-arm64/gardenlogin_darwin_arm64      | awk '{print $1}')" >> "$GITHUB_ENV"
+          echo "LINUX_SHA_AMD64=$(sha256sum bin/linux-amd64/gardenlogin_linux_amd64        | awk '{print $1}')" >> "$GITHUB_ENV"
+          echo "LINUX_SHA_ARM64=$(sha256sum bin/linux-arm64/gardenlogin_linux_arm64        | awk '{print $1}')" >> "$GITHUB_ENV"
+          echo "WINDOWS_SHA=$(sha256sum     bin/windows-amd64/gardenlogin_windows_amd64.exe | awk '{print $1}')" >> "$GITHUB_ENV"
+
+      - name: Use dummy hashes (skip‑build)
+        if: github.event_name == 'workflow_dispatch' && inputs.skip_build == true && inputs.dry_run != true
+        run: |
+          set -euo pipefail
+          dummy=dummySHA256sum
+          printf '%s\n' \
+            "DARWIN_SHA_AMD64=$dummy" \
+            "DARWIN_SHA_ARM64=$dummy" \
+            "LINUX_SHA_AMD64=$dummy"  \
+            "LINUX_SHA_ARM64=$dummy"  \
+            "WINDOWS_SHA=$dummy" >> "$GITHUB_ENV"
+
+      # ───── Homebrew tap dispatch ────────────────────────────────────────────
+      - name: Send update to ${{ github.repository_owner }}/homebrew-tap
+        if: (github.event_name == 'release' && github.event.release.prerelease == false) || (github.event_name == 'workflow_dispatch' && inputs.dry_run == false)
```

Comment:
we have this if 3 times in the file, but two times its identical and one has a different order. this should be consistent

Reply:
aligned